### DISCUSSION TITLE
在线求助，有谁知道怎么改头像吗？

### DISCUSSION THREAD

**Discussion by TOKLK** [share121/inter-knot#1982] [其他]
求助，有人知道这个绳网头像怎么改吗？

**Comment by JohnDoe-A** [comment]
点击左侧绳网头像-edit profile-再次点击头像-视角向右转找到头像图片旁的edit

**Reply by TOKLK** [reply]
谢谢，已经改好了

**Comment by Dave-12138** [comment]
点你绳网上的头像跳转到github，再点头像跳转到编辑主页，点edit，upload a photo
最近换头像问题好像有点多[1241](https://www.xn--5o0anh.top/?article=1241)
![流萤 也挺好](https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png)

**Reply by Maixiaofanqie** [reply]
直接点头像进个人主页，然后再点一下头像底下的edit（有自动翻译的就是编辑）然后upload a photo上传就行（）

**Reply by TOKLK** [reply]
谢谢，已经改好了
